schemas/common/email-interaction.schema.json

```diff
@@ -0,0 +1,136 @@
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/emailinteraction",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Email Interaction",
+  "type": "object",
+  "meta:extensible": true,
+  "description": "Captures information in the context of an email correspondence between persons",
+  "definitions": {
+    "emailinteraction": {
+      "properties": {
+        "xdm:priority": {
+          "title": "Email's priority",
+          "type": "string",
+          "description": "Captures the priority set by the person who sent the email"
+        },
+        "xdm:sentDateTime": {
+          "title": "Date and time the email was sent",
+          "type": "string",
+          "format": "date-time",
+          "description": "Captures the date and time the email correspondence was sent to an individual or a group of people"
+        },
+        "xdm:receivedDateTime": {
+          "title": "Date and time the email was received",
+          "type": "string",
+          "format": "date-time",
+          "description": "Captures the date and time when the email was received"
+        },
+        "xdm:body": {
+          "title": "Body of the mail",
+          "type": "string",
+          "description": "The contents of the email after substitution of the placeholders in the template"
+        },
+        "xdm:attachment": {
+          "title": "Email has attachments",
+          "type": "boolean",
+          "description": "A flag represent whether any attachment is part of the email message"
+        },
+        "xdm:quarantine": {
+          "title": "quarantine message",
+          "type": "boolean",
+          "description": "A flag to represent whether the email message is a spam"
+        },
+        "xdm:sizeInBytes": {
+          "title": "Email message size",
+          "type": "number",
+          "description": "Record the size of the email message in bytes"
+        },
+        "xdm:failureReasonCode": {
+          "title": "Reason code for failure",
+          "type": "string",
+          "description": "Indicates the error code for not able to deliver the email message"
+        },
+        "xdm:failureReasonMessage": {
+          "title": "Reason Message for failure",
+          "type": "string",
+          "description": "Indicates the full message for the error code, for not able to deliver the message"
+        },
+        "xdm:fromAddress": {
+          "title": "Person sending the email communication",
+          "type": "string",
+          "format": "email",
+          "description": "Person who is sender of the email message"
+        },
+        "xdm:toAddress": {
```

Comment:
If this address is an email address, we should be using the email data type here https://ns.adobe.com/xdm/context/emailaddress